crates/languages/src/rust/runnables.scm

```diff
@@ -1,3 +1,23 @@
+(
+    (attribute_item (attribute
+            (identifier) @_attribute
+            (#eq? @_attribute "cfg")
+
+            arguments: (token_tree (
+                (identifier) @_argument
+                (#eq? @_argument "test")
+            ))
+        )
+    )
+    .
+    (attribute_item) *
```

Comment:
I think we can be a bit more lenient here; We don't need to check for attributes, as it may be too restrictive at times (e.g. this query does not match with `#[cfg(any(test))]` etc).

Reply:
Do you want to just match on the `tests` identifier? So just this:
```scm
(mod_item
    name: (_) @run
    (#eq? @run "tests")
) @rust-file-test
```

Reply:
We can maybe call it `rust-mod-test` instead?

Reply:
Yup, exactly. A shorter query and rust-mod-test should suffice.